### PR TITLE
added inline validation strings for en, de and de-CH

### PR DESCRIPTION
**Pull request by kohlerdominik**
PR for #1268 

As discussed the translated strings for en, de (and de-CH).

@caouecs I made separate, non overlapping files for this. I think, maintaining them in a different branch might not be optimal. However, I'm open for other inputs where to put and how to name the files.

### CODE REVIEW

**Comment by caouecs** [comment]
Ok, thank you. Yes, it's the better idea in same branch ;)

I try to review this week, and I have to update the todo.php

File: src/en/validation-inline.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    'array'           => 'This field must be an array.',
+    'before'          => 'This must be a date before :date.',
+    'before_or_equal' => 'This must be a date before or equal to :date.',
+    'between'         => [
+        'numeric' => 'This value must be between :min and :max.',
+        'file'    => 'This file must be between :min and :max kilobytes.',
+        'string'  => 'This string must be between :min and :max characters.',
+        'array'   => 'This content must have between :min and :max items.',
+    ],
+    'boolean'        => 'This field field must be true or false.',
```
Comment (caouecs) [inline src/en/validation-inline.php:32]: field field

File: src/en/validation-inline.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    'different'      => 'This value must be different from :other.',
+    'digits'         => 'This must be :digits digits.',
+    'digits_between' => 'This must be between :min and :max digits.',
+    'dimensions'     => 'This image has invalid dimensions.',
+    'distinct'       => 'This field has a duplicate value.',
+    'email'          => 'This must be a valid email address.',
+    'ends_with'      => 'This must end with one of the following: :values.',
+    'exists'         => 'The selected value is invalid.',
+    'file'           => 'The content must be a file.',
+    'filled'         => 'This field field must have a value.',
```
Comment (caouecs) [inline src/en/validation-inline.php:46]: field field

File: src/en/validation-inline.php [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        'numeric' => 'The value must be greater than :value.',
+        'file'    => 'The file size must be greater than :value kilobytes.',
+        'string'  => 'The string must be greater than :value characters.',
+        'array'   => 'The content must have more than :value items.',
+    ],
+    'gte' => [
+        'numeric' => 'The value must be greater than or equal :value.',
+        'file'    => 'The file size must be greater than or equal :value kilobytes.',
+        'string'  => 'The string must be greater than or equal :value characters.',
+        'array'   => 'The Content must have :value items or more.',
```
Comment (caouecs) [inline src/en/validation-inline.php:57]: Content to content

File: src/en/validation-inline.php [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    'present'              => 'This field must be present.',
+    'regex'                => 'This format is invalid.',
+    'required'             => 'This field is required.',
+    'required_if'          => 'This field is required when :other is :value.',
+    'required_unless'      => 'This field is required unless :other is in :values.',
+    'required_with'        => 'This field is required when :values is present.',
+    'required_with_all'    => 'This field is required when :values are present.',
+    'required_without'     => 'This field is required when :values is not present.',
+    'required_without_all' => 'This field is required when none of :values are present.',
+    'same'                 => 'The Value of this field must match the one from :other.',
```
Comment (caouecs) [inline src/en/validation-inline.php:106]: Value to value

File: src/de-CH/validation-inline.php [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    'integer'  => 'Das muss eine ganze Zahl sein.',
+    'ip'       => 'Das muss eine gültige IP-Adresse sein.',
+    'ipv4'     => 'Das muss eine gültige IPv4-Adresse sein.',
+    'ipv6'     => 'Das muss eine gültige IPv6-Adresse sein.',
+    'json'     => 'Das muss ein gültiger JSON-String sein.',
+    'lt'       => [
+        'numeric' => 'Der Wert muss kleiner :value sein.',
+        'file'    => 'Die Datei muss kleiner :value Kilobytes gross sein.',
+        'string'  => 'Die Zeichenkette muss kleiner :value Zeichen lang sein.',
+        'array'   => ':Der Inhalt muss kleiner :value Elemente haben.',
```
Comment (caouecs) [inline src/de-CH/validation-inline.php:70]: :Der ?

File: src/de/validation-inline.php [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+    'integer'  => 'Das muss eine ganze Zahl sein.',
+    'ip'       => 'Das muss eine gültige IP-Adresse sein.',
+    'ipv4'     => 'Das muss eine gültige IPv4-Adresse sein.',
+    'ipv6'     => 'Das muss eine gültige IPv6-Adresse sein.',
+    'json'     => 'Das muss ein gültiger JSON-String sein.',
+    'lt'       => [
+        'numeric' => 'Der Wert muss kleiner als :value sein.',
+        'file'    => 'Die Datei muss kleiner als :value Kilobytes sein.',
+        'string'  => 'Die Zeichenkette muss kürzer als :value Zeichen sein.',
+        'array'   => ':Der Inhalt muss weniger als :value Elemente haben.',
```
Comment (caouecs) [inline src/de/validation-inline.php:70]: :Der ?

**Comment by caouecs** [comment]
Thank you

**Comment by kohlerdominik** [comment]
> 
> 
> Thank you

Big thanks for the intensive review and the fixes :)

**Pull request merged**